### PR TITLE
mon/OSDMonitor, mon/PaxosService: additional debug outputs when trimming

### PR DESCRIPTION
**Pull request by dvanders**
Adds some additional dout's to help troubleshooting cases where osdmaps are not trimmed.

### CODE REVIEW

File: src/mon/OSDMonitor.cc [hunk 1]
```diff
@@ -2248,22 +2248,23 @@ void OSDMonitor::share_map_with_random_osd()
 version_t OSDMonitor::get_trim_to() const
 {
   if (mon->get_quorum().empty()) {
-    dout(10) << __func__ << ": quorum not formed" << dendl;
+    dout(10) << __func__ << ": quorum not formed, trim_to = 0" << dendl;
```
Comment (tchaikov) [inline src/mon/OSDMonitor.cc:2251]: could you take the opportunity to drop the ":" to be more consistent with the rest of this file?

File: src/mon/OSDMonitor.cc [hunk 2]
```diff
@@ -2248,22 +2248,23 @@ void OSDMonitor::share_map_with_random_osd()
...
     return 0;
   }
 
   {
     std::lock_guard<std::mutex> l(creating_pgs_lock);
     if (!creating_pgs.pgs.empty()) {
+      dout(10) << __func__ << ": pgs creating, trim_to = 0" << dendl;
```
Comment (tchaikov) [inline src/mon/OSDMonitor.cc:2258]: ditto.

**Review by tchaikov — CHANGES_REQUESTED** [review]
https://github.com/ceph/ceph/pull/38070/files#r524870084

**Review by jecluis — APPROVED** [review]

**Review by tchaikov — APPROVED** [review]

**Comment by tchaikov** [comment]
- https://pulpito.ceph.com/kchai-2020-11-21_05:41:14-rados-wip-kefu-testing-2020-11-19-1104-distro-basic-smithi/
- https://pulpito.ceph.com/kchai-2020-11-21_05:41:14-rados-wip-kefu-testing-2020-11-19-1104-distro-basic-smithi/
- https://pulpito.ceph.com/kchai-2020-11-22_03:37:22-rados-wip-kefu-testing-2020-11-19-1104-distro-basic-smithi/

**Pull request merged**